packages/@storefront/core/src/services/urlUtils.ts

```diff
@@ -136,15 +137,15 @@ namespace UrlUtils {
   // tslint:disable-next-line max-line-length
   export const getAllIds = (state: Store.Indexed<Store.Navigation> | Store.AvailableNavigations, request: UrlBeautifier.SearchUrlState) => {
     return state.allIds.concat(
-      ...request.refinements.map(({ field }) => field).filter((field) => !state.allIds.includes(field))
+      ...(request.refinements || []).map(({ field }) => field).filter((field) => !state.allIds.includes(field))
```

Comment:
We don't use anything else from `request`, so `refinements` can be destructured and given a default in the function declaration.